src/migration.mjs

```diff
@@ -0,0 +1,66 @@
+import { PrismaClient } from "@prisma/client";
```

Comment:
Instead of for loops, try to use `.map()`, because then you don't have to worry about array indices